lib/process-services-cloud/src/lib/people/components/people-cloud.component.spec.ts

```diff
@@ -133,7 +133,7 @@ describe('PeopleCloudComponent', () => {
         it('should selectedUser and changedUsers emit, update selected users when a user is selected', (done) => {
             const user = { username: 'username' };
             fixture.detectChanges();
-            spyOn(component, 'isPreselectedUserValid').and.returnValue(true);
+            spyOn(component, 'isPreselectUserAlignedWithValidatedUser').and.returnValue(true);
```

Comment:
Mocking a method of CUT (class under test) is a very bad practice. It can happen, that you don't even test the component if you mock its methods. Please never mock any method of a class, what you are testing directly. Only collaborator classes could be mocked.